neutrinordp/xrdp-neutrinordp.c

```diff
@@ -504,9 +516,13 @@ lxrdp_set_param(struct mod *mod, const char *name, const char *value)
     {
         settings->desktop_resize = g_text2bool(value);
     }
-    else if (g_strcmp(name, "pamusername") == 0 ||
-             g_strcmp(name, "pampassword") == 0 ||
-             g_strcmp(name, "pammsessionmng") == 0)
+    else if (g_strcmp(name, "pamusername") == 0)
+    {
+        g_strncpy(mod->pamusername, value, 255);
+    }
+    else if (g_strcmp(name, "pampassword") == 0 ||
+             g_strcmp(name, "pamsessionmng") == 0 ||
+             g_strcmp(name, "delay_ms") == 0)
```

Comment:
See comment below about `delay_ms` - this shouldn't be here.

Reply:
Fixed in #1f09ad9.